pkg/app/build.go

```diff
@@ -89,6 +90,11 @@ To build and push the image to a registry:
 			Usage:   "Import the cache (e.g. type=registry,ref=<image>)",
 			Aliases: []string{"ic"},
 		},
+		&cli.StringFlag{
+			Name:        "platform",
+			Usage:       "Specify the target platforms for the build output (for example, windows/amd64 or linux/amd64,darwin/arm64)",
+			DefaultText: "linux/amd64",
```

Comment:
I wonder if we should use the local host platform as the default one.

Reply:
Ok, I will use the GOOS/GOARCH as the default one